### ISSUE TITLE
false positive Error LC0003/LC0014 for NextMajor [BC26, Sandbox]

### ISSUE THREAD

**Issue by ralf-escher** [StefanMaron/BusinessCentral.LinterCop#886]
I set rule `LC0003` to been shown as an error.
This works fine for my BC25 Azure DevOps pipeline, but shows now many errors for my NextMajor pipeline run:

it's always the same issue, AccessByPermission = TableData xxx on a Page is shown as an error

`page/AutomotiveSalesOrder.Page.al(319,42): error LC0003: Do not use an Object ID for properties or variables declaration. 
Use TableData "Responsibility Center" = R instead.`

However this Page uses the correct code:

```
field("Responsibility Center"; Rec."Responsibility Center")
{
    AccessByPermission = TableData "Responsibility Center" = R;
    ApplicationArea = Basic, Suite;
}
```

Any idea?
Same problem like in #824 ?

**Comment by Arthurvdv** [comment]
>  NextMajor pipeline run

Which assets/artifact of the `BusinessCentral.LinterCop.dll` are you using in your pipeline?

**Comment by ralf-escher** [comment]
I am using 
[https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/latest/download/BusinessCentral.LinterCop.dll](url)

**Comment by Arthurvdv** [comment]
Could you try this?
https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/latest/download/BusinessCentral.Pre-Release.LinterCop.dll

The Next Major of Business Central, currently version 26.0.29633.0, uses the AL Language version 15.0.1300427. The current release version of AL is 14.2.1249978, where the BusinessCentral.LinterCop.dll file is compiled against 14.2.1249978.

![Image](https://github.com/user-attachments/assets/2c720d5a-1934-4ac1-a22f-9c8fdb4fba0b)

The LinterCop.dll needs to be compiled against the exact same version of the AL Language in order to work, otherwise unexpected errors and false positives occur.

**Comment by ralf-escher** [comment]
It's easier to set the (wrong) error to a warning and to wait until Apri/May for the next release.
Starting from last night I also get a wrong positive error `LC0014` for NextMajor.

**Comment by ralf-escher** [comment]
@Arthurvdv:
I have meanwhile set up a completely new project in GitHub with AL-Go and again encountered issues when validating against NextMajor and NextMinor.

The following configuration for `Test Next Major.settings.json` (within AL-Go) worked:
https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/latest/download/BusinessCentral.LinterCop.next.dll

**Comment by Arthurvdv** [comment]
We'll continue this conversion in https://github.com/StefanMaron/BusinessCentral.LinterCop/issues/1095
